Dockerfile

```diff
@@ -19,11 +17,11 @@ RUN \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
-RUN pip3 install pipenv==2023.12.1
+RUN pip3 install --upgrade pdm
```

Comment:
If you install only prod requirements, you don't need PDM, because runtime dependencies, PEP 621, is a standard that pip undestands

Reply:
Oh, that's a great point! Going to do this.